apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -222,6 +272,18 @@ const BookingPage = ({
     }
   };
 
+  // Calculate the booking date(s)
```

Comment:
This is to be able to iterate through the dates later with their string representation as well as their Date representation